shared/modules/macros.jinja

```diff
@@ -0,0 +1,51 @@
+
+{{# Example usage: sshd_check(default="no", option="Banner", value="/etc/issue") #}}
+{{% macro sshd_check(default, option, value) -%}}
+|-
```

Comment:
This `|-` appears in `ssg-rhel7-ocil.xml`

Reply:
Nice catch, I have replaced it with a pipe alone as indicated here: https://stackoverflow.com/questions/15540635/what-is-the-use-of-pipe-symbol-in-yaml